.travis.yml

```diff
@@ -11,21 +11,17 @@ before_script:
     - shopt -s extglob
     - cp ./!(build-copy) build-copy/aws-mock-copy/ -r
     - cd build-copy/aws-mock-copy
-    - nohup gradle jettyRun &
     - cd ../..
     - node example/nodejs/LocalProxy.js &
-    - sleep 5
 
 script: 
-    - gradle clean coberturaCheck Javadoc check
-    - npm test
```

Comment:
This is also offered as an npm package, so we have to make sure that the npm test runs: https://www.npmjs.com/package/aws-mock

Reply:
Yes, this always runs. Instead of running this task separately in Travis, it now runs as part of integrationTest task via Gradle. So, we will always run this check as part of build.